CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 - Disable anonymous reports in kong default values for proxy and ingress controller.
 - Align with upstream chart version [2.35.1](https://github.com/Kong/charts/releases/tag/kong-2.35.1) ([Changes in upstream repository](https://github.com/Kong/charts/compare/kong-2.33.3...kong-2.35.1))
   - Kong listens now use both IPv4 and IPv6 addresses. [kong/charts#986](https://github.com/Kong/charts/pull/986)
+- Update kong ingress controller to [3.1.0](https://github.com/Kong/kubernetes-ingress-controller/blob/v3.1.0/CHANGELOG.md#310)
```

Comment:
This version adds an admission webhook to `HTTPRoute` CRs. We need to evaluate if this clashes with linkerd